contrib/gitian-build.sh

```diff
@@ -23,10 +23,6 @@ ossTarHash="f9a8cdb38b9c309326764ebc937cba1523a3a751a7ab05df3ecc99d18ae466c9"
 macosSdkUrl="https://depends.dogecoincore.org/MacOSX10.11.sdk.tar.gz"
```

Comment:
10.11 is super long in the tooth at this point and is going to start to run into functionality issues soon -- but since it's out of scope for this review i'm just leaving a comment to keep it in everyone's mind.